components/ml/org.wso2.carbon.ml.core/src/main/java/org/wso2/carbon/ml/core/spark/algorithms/SupervisedSparkModelBuilder.java

```diff
@@ -116,46 +125,46 @@ public MLModel build() throws MLModelBuilderException {
-                        mlModel, includedFeatures);
-                break;
-            case LASSO_REGRESSION:
-                summaryModel = buildLassoRegressionModel(sparkContext, modelId, trainingData, testingData, workflow,
-                        mlModel, includedFeatures);
-                break;
-            default:
-                throw new AlgorithmNameException("Incorrect algorithm name");
+                case LOGISTIC_REGRESSION:
+                    summaryModel = buildLogisticRegressionModel(sparkContext, modelId, trainingData, testingData, workflow,
+                            mlModel, includedFeatures, true);
+                    break;
+                case LOGISTIC_REGRESSION_LBFGS:
+                    summaryModel = buildLogisticRegressionModel(sparkContext, modelId, trainingData, testingData, workflow,
+                            mlModel, includedFeatures, false);
+                    break;
+                case DECISION_TREE:
+                    categoricalFeatureInfo = getCategoricalFeatureInfo(context.getEncodings());
+                    summaryModel = buildDecisionTreeModel(sparkContext, modelId, trainingData, testingData, workflow,
+                            mlModel, includedFeatures, categoricalFeatureInfo);
+                    break;
+                case RANDOM_FOREST:
+                    categoricalFeatureInfo = getCategoricalFeatureInfo(context.getEncodings());
+                    summaryModel = buildRandomForestTreeModel(sparkContext, modelId, trainingData, testingData, workflow,
+                            mlModel, includedFeatures, categoricalFeatureInfo);
+                    break;
+                case SVM:
+                    summaryModel = buildSVMModel(sparkContext, modelId, trainingData, testingData, workflow, mlModel,
+                            includedFeatures);
+                    break;
+                case NAIVE_BAYES:
+                    summaryModel = buildNaiveBayesModel(sparkContext, modelId, trainingData, testingData, workflow,
+                            mlModel, includedFeatures);
+                    break;
+                case LINEAR_REGRESSION:
+                    summaryModel = buildLinearRegressionModel(sparkContext, modelId, trainingData, testingData, workflow,
+                            mlModel, includedFeatures);
+                    break;
+                case RIDGE_REGRESSION:
+                    summaryModel = buildRidgeRegressionModel(sparkContext, modelId, trainingData, testingData, workflow,
+                            mlModel, includedFeatures);
+                    break;
+                case LASSO_REGRESSION:
+                    summaryModel = buildLassoRegressionModel(sparkContext, modelId, trainingData, testingData, workflow,
+                            mlModel, includedFeatures);
+                    break;
+                default:
+                    throw new AlgorithmNameException("Incorrect algorithm name");
```

Comment:
Please don't format irrelevant code with a different purpose commit, as it clutters your real fix and makes it hard to review. Are you using WSO2 code template?